deps/graph-parser/test/logseq/graph_parser_test.cljs

```diff
@@ -228,8 +217,7 @@
                 (map #(select-keys % [:block/properties :block/invalid-properties]))))
         "Has correct (in)valid block properties")
 
-    (is (= [{:block/properties {:foo "valid"}
-             :block/invalid-properties #{"[[foo]]" "some,prop"}}]
```

Comment:
I think we should preserve the previous behavior. If page properties don't have the same validity as block properties this inconsistency can be confusing and lead to bugs for queries. For example, `page-property` filter will return back valid properties that are invalid for `block-property`

Reply:
Updated in [b0aecd1](https://github.com/logseq/logseq/pull/6529/commits/b0aecd10b47eca7297d6ff11df687f5692791fe8).